pkg/exchange/okex/stream_test.go

```diff
@@ -0,0 +1,121 @@
+package okex
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/c9s/bbgo/pkg/exchange/okex/okexapi"
+	"github.com/c9s/bbgo/pkg/testutil"
+	"github.com/c9s/bbgo/pkg/types"
+)
+
+func getTestClientOrSkip(t *testing.T) *Stream {
+	if b, _ := strconv.ParseBool(os.Getenv("CI")); b {
+		t.Skip("skip test for CI")
+	}
+
+	key, secret, passphrase, ok := testutil.IntegrationTestWithPassphraseConfigured(t, "OKEX")
+	if !ok {
+		t.Skip("Please configure all credentials about OKEX")
+		return nil
+	}
+	e := New(key, secret, passphrase)
+
+	return NewStream(e.client)
+}
+
+func TestStream(t *testing.T) {
+	t.Skip()
+	s := getTestClientOrSkip(t)
+
+	t.Run("Auth test", func(t *testing.T) {
+		s.Connect(context.Background())
+		c := make(chan struct{})
+		<-c
```

Comment:
The channel `c` is created within the inner function scope, and nobody access the `c`. Could you add a comment to describe why we have to wait here?

Reply:
to wait for websocket server push event to client, from my experience of testing this file, I think so.

Reply:
1. All tests only test whether we receive an error from `Connect` method, and other callbacks only print messages for human instead of asserting something. 

2. We are blocked by `<-c` forever. How do we know we have connected to the websocket server?

Reply:
The answer is like below, I originally thought unit test is just helpful for debugging in local, if we do test it in jenkins flow, we should really verify something (format, etc..) or just not commit testing files.

Reply:
The `<-c` also creates a deadlock here. That means the process is blocked forever until we terminate it. It's a serious issue.

<img width="460" alt="image" src="https://github.com/c9s/bbgo/assets/1477617/f38522af-4137-4344-86bb-412a80711f35">

Reply:
Updated, So far I came up a new way to test stream. Maybe not the best, but I avoid the issue. But you can just focus on https://github.com/c9s/bbgo/pull/1333